debian/patches/fix-cgfs_types

```diff
@@ -0,0 +1,61 @@
+Description: <short summary of the patch>
```

Comment:
Should we not change this and the other fields below?

Reply:
Yeah we should, but I don't really see the use as our package misses quite a lot of information. But if you want I can add it.

Reply:
Fair enough. Let's do that in another TR then and just fix it all at once